juspay/hyperswitch
# refactor(core):  Refactor fallback routing behaviour in payments for v2

## Type of Change



## Description
This PR refactors the  fallback routing behaviour in payments routing for v2. Update the function to accept business_profile as a parameter instead of profile_id.


### Additional Changes





## Motivation and Context



## How did you test it?
For v2:
- create a MA and add 2 MCAs, it will be added for default fallback in the order they were cretaed 
- make a payment , it should go through the mca that was added first based on default fallback


## Checklist
